### ISSUE TITLE
Port to Python 3

### ISSUE THREAD

**Issue by dorellang** [taschini/pyinterval#3]
I tried to install this library in Python 3 and I got this error.

```
Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-build-j9ciam2g/pyinterval/setup.py", line 58, in <module>
        @apply
    NameError: name 'apply' is not defined
```

Then I checked the package info. in PyPI and I saw that this is available only for Python 3. Would it be too much trouble to port the library to Python 3? I could help you to do it...

Cheers.

**Comment by taschini** [comment]
That's a good idea. I'll start working on it the coming week.

**Comment by taschini** [comment]
I created [PyCRlibm](/taschini/pycrlibm) as a spin-off of this project to host the C extension with the bindings to CRlibm.

The master branch in PyCRlibm now supports Python 3, and I'll soon prepare a release. Then I'll look into PyInterval proper.

**Comment by taschini** [comment]
The master branch now supports Python 3, and is tested with Python 3.5.

I'll update the docs and then I'll prepare a new release on PyPI.

**Comment by taschini** [comment]
I just released version 1.1.0 on [PyPI](https://pypi.python.org/pypi/pyinterval), which supports Python 3.
